README.md

```diff
@@ -41,7 +41,7 @@ You will find detailed guides and frequently asked questions there.
 
 ### Configuring Bitcoin Core
 
-:warning: Eclair requires Bitcoin Core 0.18.1 or 0.19.1. If you are upgrading an existing wallet, you need to create a new address and send all your funds to that address.
+:warning: Eclair requires Bitcoin Core 0.19.1 or 0.20.1. If you are upgrading an existing wallet, you need to create a new address and send all your funds to that address.
```

Comment:
Any reason why we drop the 0.18.1 compatibility?

Reply:
Not really, we could still support it as we haven't done anything breaking. If you prefer I can change that to 0.18.1, 0.19.1 or 0.20.1.
I'm mostly waiting for release 0.21 that will add a `psbtbumpfee` RPC, which is required for automatic fee bumping of anchor output channels, but for now we don't need to deprecate support.

Reply:
I would say if we don't need to, let's not break compatibility.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1526/commits/b8170cd7c5bdf04e49b10350deb9ff52e19b06a7